frontend/app/views/index.scala.html

```diff
@@ -8,7 +8,7 @@
 @import views.support.NavAnchor
 @import views.support.Asset
 
-@main("Join", pageInfo) {
+@main("", pageInfo) {
```

Comment:
@afiore Weirdly `title` is **already** in PageInfo, so not sure why we include it here too tbh.